### DISCUSSION TITLE
Download csv file

### DISCUSSION THREAD

**Discussion by ghost** [aceberg/WatchYourLAN#145] [Ideas]
It would be great to add a  button to download a csv file with all the IP/mac/Hostname infos. I use NetAlert to do this but WatchYourLAN is much simpler and cleaner to get an overview of network.

**Comment by aceberg** [comment]
There is no such button right now, but you can use `/api/all` to get this data in `json`.

**Comment by ghost** [comment]
Thanks for the tip.
